crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -455,6 +456,32 @@ def services
                   node["pacemaker"]["founder"] == node[:fqdn])
             nova_node = node
           end
+          if node.roles.include?("database-server") &&
+              (!node.roles.include?("pacemaker-cluster-member") ||
+                  node["pacemaker"]["founder"] == node[:fqdn])
+            database_node = node
+          end
+        end
+
+        begin
+          unless database_node.nil?
+            database_node.wait_for_script_to_finish(
+              "/usr/sbin/crowbar-dump-monasca-db.sh",
```

Comment:
This script is generated only when `monasca_enabled==true`. There should be some guard code to avoid trying to execute non-existing script.

Reply:
Done.